python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -127,6 +128,12 @@ class ProcessBrightStarsConfig(pipeBase.PipelineTaskConfig,
             " annular flux.",
         default=('BAD', 'CR', 'CROSSTALK', 'EDGE', 'NO_DATA', 'SAT', 'SUSPECT', 'UNMASKEDNAN')
     )
+    minPixelsWithinFrame = pexConfig.Field(
+        dtype=int,
+        doc="When a bright star is beyond exposure boundary, what is the minimal number of pixels that still "
```

Comment:
I would suggest rewording to something like: The minimum number of pixels that must fall within the stamp boundary for the bright star to be saved when it is beyond the exposure boundary.